OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -96,6 +119,103 @@ public void onPageLoadProgressChanged(WebView webView, int progress) {
         }
     }
 
+    private void fetchDateBannerInfo() {
+        if (canFetchBannerInfo && unit.getType() == BlockType.PROBLEM) {
+            courseDateViewModel = new ViewModelProvider(this, new ViewModelFactory()).get(CourseDateViewModel.class);
```

Comment:
should be moved in `initObserver` to avoid the init crash

Reply:
done